src/MUIDataTable.js

```diff
@@ -259,6 +260,8 @@ class MUIDataTable extends React.Component {
   componentDidMount() {
     this.setHeadResizeable(this.headCellRefs, this.tableRef);
 
+    this.setState({ pagination: this.options.pagination });
```

Comment:
I don't understand this addition with regards to the printing update.

Reply:
I forgot to mention it int PR's description, but I'm hiding pagination too. For reports is important to print the whole data, not only the current pagination.